docs/index.markdown

```diff
@@ -0,0 +1,27 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: home
+title: About
+nav_order: 1
+---
+
+# CF Node.js Logging Support
+
+[![Version npm](https://img.shields.io/npm/v/cf-nodejs-logging-support.svg)](https://www.npmjs.com/package/cf-nodejs-logging-support)
+[![npm Downloads](https://img.shields.io/npm/dm/cf-nodejs-logging-support.svg)](https://www.npmjs.com/package/cf-nodejs-logging-support)
+[![Build Status](https://travis-ci.com/SAP/cf-nodejs-logging-support.svg?branch=master)](https://travis-ci.com/SAP/cf-nodejs-logging-support)
+[![REUSE status](https://api.reuse.software/badge/github.com/SAP/cf-nodejs-logging-support)](https://api.reuse.software/info/github.com/SAP/cf-nodejs-logging-support)
+
+This is a collection of support libraries for node.js applications running on Cloud Foundry that serve two main purposes: 
+It provides (a) means to emit structured application log messages and (b) instrument parts of your application stack to collect request metrics.
+
+To get started follow our [Getting Started](/cf-nodejs-logging-support/getting-started/) section to install the library and learn how to integrate it with supported server frameworks.
+
+Head over to the [General Usage](/cf-nodejs-logging-support/general-usage) articles to find information about the main logging functions. 
+
+Advanced topics, e.g., dynamic logging level threshold, SAP Passport support get covered in [Advanced Usage](/cf-nodejs-logging-support/advanced-usage) articles.
```

Comment:
the advanced usage

Reply:
I would go for "More advanced topics ...".
Otherwise it sounds like we reference them again after introducing them somewhere above, which we haven't.